8_visualize_metrics/src/plot_metric_map .R

```diff
@@ -33,5 +33,5 @@ plot_metric_map <- function(out_file, network_ind, metric_file, metric_col, plot
     map_theme +
     labs(color = legend_text)
 
-  ggsave(out_file, p, height = 6, width = 6)
+  ggsave(out_file, p, height = 7.5, width = 13)
```

Comment:
Does this actually produce a 13 inch wide figure? Then that's probably a lot of white space and unnecessary. You might be able to get away with only specifying height, and will produce whatever width it needs to keep the ratio of the plot correct.